examples/gno.land/p/demo/simpledao/propstore.gno

```diff
@@ -63,15 +70,20 @@ func (p *proposal) Render() string {
 	// Fetch the voting stats
 	stats := p.Stats()
 
-	output := ""
-	output += ufmt.Sprintf("Author: %s", p.Author().String())
-	output += "\n\n"
-	output += p.Description()
-	output += "\n\n"
-	output += ufmt.Sprintf("Status: %s", p.Status().String())
-	output += "\n\n"
-	output += ufmt.Sprintf(
-		"Voting stats: YES %d (%d%%), NO %d (%d%%), ABSTAIN %d (%d%%), MISSING VOTE %d (%d%%)",
+	var out bytes.Buffer
+
+	out.WriteString("## Description\n\n")
```

Comment:
```suggestion
```

no need for h2 description, it's the description, just after the title.

Reply:
rm'd the H2

Reply:
Honestly, it looks better with the H2. I'd like to keep it.

With:
<img width="915" alt="Screenshot 2024-11-21 at 12 56 45" src="https://github.com/user-attachments/assets/463eab59-d9ca-4f2c-a331-9100b202c7e1">

Without:

<img width="995" alt="Screenshot 2024-11-21 at 12 58 03" src="https://github.com/user-attachments/assets/0c90c6c7-a73b-48a6-a1d7-6cd2ea40b8a5">